src/sentry/conf/server.py

```diff
@@ -1124,6 +1126,10 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
         # Run every 20 minutes
         "schedule": crontab(minute="*/20"),
     },
+    "detect-regressed-functions": {
```

Comment:
nit: should we rename this to something more generic since the task will run regressions on both transactions and functions